src/apic-extension/azext_apic_extension/_help.py

```diff
@@ -31,4 +31,7 @@
         text: |
           az apic api register -g api-center-test -n contosoeuap --api-location "examples/cli-examples/spec-examples/openai.json" --environment-id public
           az apic api register -g api-center-test -n contosoeuap --api-location "examples/cli-examples/spec-examples/openai.yml" --environment-id public
+      - name: Register api by providing spec url.
+        text: |
+          az apic api register -g api-center-test -n contosoeuap --api-location "https://petstore.swagger.io/v2/swagger.json" --environment-id public
```

Comment:
Do we need to add `az apic api register -g api-center-test -n contosoeuap --api-location "https://petstore.swagger.io/v2/swagger.yaml" --environment-id public` to show that both yaml and json format from url can be supported